### PR TITLE
Use new grpc endpoints from blobber

### LINKED ISSUE #44: Use new grpc endpoints from blobber

### LINKED ISSUE #66: Integrating grpc equivalent of "/v1/file/objecttree/{allocation}" of blobber

### LINKED ISSUE #67: Integrating grpc equivalent of "/allocation" of blobber

### LINKED ISSUE #68: Integrating grpc equivalent of "/v1/file/meta/{allocation}" of blobber

### LINKED ISSUE #69: Integrating grpc equivalent of "/v1/file/stats/{allocation}" of blobber

### LINKED ISSUE #70: Integrating grpc equivalent of "/v1/file/list/{allocation}" of blobber

### LINKED ISSUE #71: Integrating grpc equivalent of "/v1/file/objectpath/{allocation}" of blobber

### LINKED ISSUE #72: Integrating grpc equivalent of "/v1/file/referencepath/{allocation}" of blobber

### PR DESCRIPTION
**Pull request by shravanshetty1**
closes #44 
closes #66
closes #67
closes #68
closes #69
closes #70
closes #71
closes #72

### CODE REVIEW

File: core/clients/blobberClient/client.go [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+	blobbercommon "github.com/0chain/blobber/code/go/0chain.net/core/common"
+	"github.com/0chain/gosdk/zboxcore/client"
+	"google.golang.org/grpc/metadata"
+
+	"github.com/0chain/blobber/code/go/0chain.net/blobbercore/blobbergrpc"
+	"google.golang.org/grpc"
+)
+
+const GRPCPort = 7031
```
Comment (shravanshetty1) [inline core/clients/blobberClient/client.go:22]: This probably requires a design discussion

Comment (shravanshetty1) [reply core/clients/blobberClient/client.go:22]: We are currently using a fixed grpc port, this should be configurable by the blobber. Blobber should provide the grpc addr its listening on, currently it provides only http addr - this is a large task and is best handled in another issue. https://github.com/0chain/gosdk/issues/52

File: zboxcore/sdk/filemetaworker.go [hunk 2]
```diff
@@ -27,60 +24,42 @@ type fileMetaResponse struct {
...
 	defer req.wg.Done()
-	body := new(bytes.Buffer)
-	formWriter := multipart.NewWriter(body)
 
 	var fileRef *fileref.FileRef
 	var s strings.Builder
 	var err error
+	grpcReq := &blobbergrpc.GetFileMetaDataRequest{
+		Allocation: req.allocationTx,
+	}
```
Comment (Sriep) [inline zboxcore/sdk/filemetaworker.go:33]: Is there a convenient way of mocking `blobbergrpc` functions for unit tests? For instance PR https://github.com/0chain/gosdk/pull/48 has a unit test for `getFileMetaInfoFromBlobber` and will need to merge this change in.

Comment (shravanshetty1) [reply zboxcore/sdk/filemetaworker.go:33]: will need to run mockery in blobber to generate mocks for https://github.com/0chain/blobber/blob/a73ef1703eb84fea60bfe48e5771b99978ec13b0/code/go/0chain.net/blobbercore/blobbergrpc/blobber_grpc.pb.go#L19
Then we import `blobbercore/mocks` here

Comment (shravanshetty1) [reply zboxcore/sdk/filemetaworker.go:33]: @Sriep

Comment (Sriep) [reply zboxcore/sdk/filemetaworker.go:33]: Looks ok. Do we need this merged?

Comment (shravanshetty1) [reply zboxcore/sdk/filemetaworker.go:33]: Its being merged to grpc not master. There will be a seperate PR between master<>grpc so issues will be caused then - ofcourse current grpc branch will be tested on a devnet before we merge it on master.

I need to make some go.mod changes before i merge it so - if you approve this changes please add your approval, thank you!

Comment (shravanshetty1) [reply zboxcore/sdk/filemetaworker.go:33]: Added an issue - https://github.com/0chain/blobber/issues/176

**Review by Sriep — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -32,6 +32,12 @@ sdkver:
 gosdk-test:
 	go test -tags bn256 ./...
 
+gosdk-test:
```
Comment (shravanshetty1) [inline Makefile:35]: duplicate?

**Pull request merged**